### ISSUE TITLE
-----> Fetching custom git buildpack... done   !     Push rejected, no Cedar-supported app detected

### ISSUE THREAD

**Issue by OKNoah** [mcollina/heroku-buildpack-graphicsmagick#16]
I get this error deploying to Heroku for the first time after setting the build pack.

    -----> Fetching custom git buildpack... done

     !     Push rejected, no Cedar-supported app detected

**Comment by benalavi** [comment]
@OKNoah What buildpacks are you using?

**Comment by OKNoah** [comment]
I hadn't followed the `.buildpack` part, I think.

**Comment by dragoninfrontofme** [comment]
HI,

I got similar issue, 
my .buildpacks are:
https://github.com/mcollina/heroku-buildpack-graphicsmagick
https://github.com/heroku/heroku-buildpack-scala

any idea?

**Comment by benalavi** [comment]
@dragoninfrontofme Does it work if you use only the scala buildpack?

**Comment by dragoninfrontofme** [comment]
@benalavi  no, it didn't work either. I tried only scala buildpack, let me try re-install everything again from the beginning, it seems heroku not able identify my ".buildpacks" file

**Comment by benalavi** [comment]
@dragoninfrontofme https://github.com/heroku/heroku-buildpack-scala#how-it-works describes how Heroku detects your Scala app, based on the error I would guess that it's not detecting your app at all

**Comment by dragoninfrontofme** [comment]
@benalavi i successfully installed the buildpacks on my app, but forgive my silly question, how to use the graphicmagick on my app?
Currently im using playframework - scala, im using im4java library which is looking for "gm", i got error that can't find "gm"
